Formula/arangodb.rb

```diff
@@ -45,26 +48,28 @@ def install
     mkdir "build" do
       args = std_cmake_args + %W[
         -DHOMEBREW=ON
-        -DUSE_OPTIMIZE_FOR_ARCHITECTURE=OFF
-        -DASM_OPTIMIZATIONS=OFF
-        -DCMAKE_INSTALL_DATADIR=#{share}
-        -DCMAKE_INSTALL_DATAROOTDIR=#{share}
-        -DCMAKE_INSTALL_SYSCONFDIR=#{etc}
-        -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
-        -DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo
+        -DUSE_MAINTAINER_MODE=Off
         -DUSE_JEMALLOC=Off
+        -DCMAKE_SKIP_RPATH=On
+        -DOPENSSL_USE_STATIC_LIBS=On
+        -DCMAKE_LIBRARY_PATH=#{prefix}/opt/openssl@1.1/lib
+        -DOPENSSL_ROOT_DIR=#{prefix}/opt/openssl@1.1/lib
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}
+        -DTARGET_ARCHITECTURE=nehalem
+        -DUSE_CATCH_TESTS=Off
+        -DUSE_GOOGLE_TESTS=Off
+        -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
       ]
 
       if ENV.compiler == "gcc-6"
         ENV.append "V8_CXXFLAGS", "-O3 -g -fno-delete-null-pointer-checks"
       end
 
+      ENV.append "CXXFLAGS", "-std=c++14"
```

Comment:
Why is this not detected by the build system?

Reply:
removed. Maybe this was necessary before I removed the link ENV.cxx11